### PR TITLE
XRays: Support colormaps besides Jet

### PR DESCRIPTION
**Pull request by nnmm**
This allows using a different colormap that interpolates between a light blue and black for stddev xrays.

Should we also generalize regular XRays and Intensity-colored xrays with colormaps?

### CODE REVIEW

**Comment by nnmm** [comment]
ptal @feuerste

File: xray/src/generation.rs [hunk 1]
```diff
@@ -96,6 +49,15 @@ arg_enum! {
     }
 }
 
+arg_enum! {
+    #[derive(Debug)]
+    #[allow(non_camel_case_types)]
+    pub enum ColormapArgument {
+        jet,
+        purpley,
```
Comment (feuerste) [inline xray/src/generation.rs]: maybe `purplish`?

Comment (tblaette) [reply xray/src/generation.rs]: how about `purplig`? ;-)

Comment (feuerste) [reply xray/src/generation.rs]: Haha, honestly, purplish exists, the others don't, so I would opt for the existing one...

File: xray/src/generation.rs [hunk 2]
```diff
@@ -115,9 +78,21 @@ impl ColoringStrategyKind {
...
-                Box::new(HeightStddevColoringStrategy::new(max_stddev))
+            ColoringStrategyKind::ColoredWithHeightStddev(max_stddev, ColormapArgument::jet) => {
+                Box::new(HeightStddevColoringStrategy::<Jet>::new(max_stddev, Jet {}))
             }
+            ColoringStrategyKind::ColoredWithHeightStddev(
+                max_stddev,
+                ColormapArgument::purpley,
+            ) => Box::new(HeightStddevColoringStrategy::new(
+                max_stddev,
+                Monochrome(Color {
```
Comment (feuerste) [inline xray/src/generation.rs]: How about defining a `const` for this color?

**Pull request merged**